NEWS.md

```diff
@@ -1,3 +1,86 @@
-# Changes for Melange 0.1
+# Major changes from 0.1.0 to 0.2.0
+
+* Added experimental support for running containers using Docker,
+  rather than Bubblewrap.
+
+* Added initial support for generating SBOMs for each package.
+  These SBOMs will be used with other composition tools to generate
+  so-called multi-layer SBOMs.
+
+* Added support for breakpoint labels in pipelines.  These allow
+  a build to be suspended or resumed from a specific point in the
+  pipeline.  Care must be used to ensure the build environment
+  locations are supplied when resuming a build, or a new build
+  environment will be created.
```

Comment:
This is all great. The only suggestion I have is to change the part here that says "Care must be used..." to something like "In order to use this feature, a build environment location must be provided when resuming a build, otherwise a new build environment will be created."